### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-docker.git`](https://github.com/cisagov/skeleton-docker.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/con-pca-api.git con-pca-api
    cd con-pca-api
    git remote add skeleton https://github.com/cisagov/skeleton-docker.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .pre-commit-config.yaml docker-compose.yml tests/container_test.py 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: .github/dependabot.yml
Comment (jsf9k) [inline .github/dependabot.yml]: These lines should be uncommented.

**Review by jsf9k — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml
Comment (itsmostafa) [reply .github/dependabot.yml]: ah, i forgot this again. resolved with: [8a8482e](https://github.com/cisagov/con-pca-api/pull/637/commits/8a8482e07d79753227646172c44ce856ca7814f2)

File: .github/dependabot.yml
Comment (jsf9k) [inline .github/dependabot.yml:2]: There is no reason to remove this blank line that comes from the skeleton.

**Review by jsf9k — APPROVED** [review]
You removed one line too many, I think.

File: .github/dependabot.yml
Comment (itsmostafa) [reply .github/dependabot.yml:2]: @jsf9k resolved with [04b5d49](https://github.com/cisagov/con-pca-api/pull/637/commits/04b5d4920cb3ec0007f909063c3f407a92b660d1)

**Comment by dav3r** [comment]
FYI, we strongly recommend requiring 2 approvals before merging PRs.

**Pull request merged**
